src/evaluators/interpolation/PHAL_DOFCellToSide.hpp

```diff
@@ -59,14 +62,66 @@ class DOFCellToSideBase : public PHX::EvaluatorWithBaseImpl<Traits>,
     CELL_VECTOR,
     CELL_TENSOR,
     NODE_SCALAR,
+    NODE_SCALAR_SIDESET,
     NODE_VECTOR,
+    NODE_VECTOR_SIDESET,
     NODE_TENSOR,
-    VERTEX_VECTOR
+    VERTEX_VECTOR,
+    VERTEX_VECTOR_SIDESET
   };
 
   LayoutType layout;
 
   MDFieldMemoizer<Traits> memoizer;
+
+  Kokkos::DynRankView<int, PHX::Device> dimsView;
+
+public:
+
+  typedef Kokkos::View<int***, PHX::Device>::execution_space ExecutionSpace;
```

Comment:
You can get the exec space out of `PHX::Device` directly, keeping the code more readable.

Reply:
Also, and this is another 'style/taste' comment, why not using...using (sorry for the pun)? Like

`using ExecutionSpace = ...;`

It's the C++11 way, and I think it's easier to read. Again, this might be a taste issue, so feel free to ignore.

Reply:
I think we can do a large sweep and change all instances of this to `using ExecutionSpace = PHX::Device::execution_space` later.